controllers/controllers/services/cfservicebinding_controller_test.go

```diff
@@ -185,7 +196,7 @@ var _ = Describe("CFServiceBinding.Reconcile", func() {
 					})
 					It("it creates a servicebinding.io ServiceBinding with the type/provider filled in", func() {
 						Expect(fakeClient.CreateCallCount()).To(Equal(1), "Client.Create call count mismatch")
-						Expect(fakeClient.PatchCallCount()).To(Equal(1), "Client.Patch call count mismatch")
+						Expect(fakeClient.PatchCallCount()).To(Equal(2), "Client.Patch call count mismatch")
```

Comment:
Should the test not check the arguments to the patch operation in addition to checking the count? Why is the call count important but calls' args are not?

Reply:
The 2 patch function calls sets the owner ref and updates the VCAP services secret. The patch functionality is actually tested in the integration test. That verifies the owner refs are set and the VCAP services secret is updated. We didn't want to duplicate the same testing conditions in the unit test as well.